packages/core/src/browser/keybinding.ts

```diff
@@ -168,36 +168,36 @@ export class KeybindingRegistry {
      *
      * Keybindings registered later have higher priority during evaluation.
      *
-     * @param binding
+     * @param binding the {@link common.Keybinding} to be registered
```

Comment:
I'm not sure the `{@link}` tags are necessary when describing `params`, developers can view the definition of parameters directly from the signature of the method, and can always `go to definition` to see it's entire definition.

In addition, I mentioned earlier that `{@link}` tags do not work yet so it looks noisy and does not render correctly.

Reply:
I agree, I removed all links from parameters, if the parameter type was the linked type